src/main/java/org/jabref/gui/preview/PreviewPanel.java

```diff
@@ -167,4 +154,20 @@ public void setEntry(BibEntry entry) {
     public void print() {
         previewView.print();
     }
+
+    public void nextPreviewStyle() {
+        cyclePreview(Globals.prefs.getPreviewPreferences().getPreviewCyclePosition() + 1);
+    }
+
+    public void previousPreviewStyle() {
+        cyclePreview(Globals.prefs.getPreviewPreferences().getPreviewCyclePosition() - 1);
+    }
+
+    private void cyclePreview(int newPosition) {
+        PreviewPreferences previewPreferences = Globals.prefs.getPreviewPreferences()
+                .getBuilder()
+                .withPreviewCyclePosition(newPosition)
+                .build();
+        updateLayout(previewPreferences);
```

Comment:
Is there no need to store the new layout in the preferences? (Should probably be done in updateLayout)

Reply:
Yes, good catch!